src/main/java/Duke.java

```diff
@@ -1,10 +1,133 @@
+import java.util.Random;
+import java.util.Scanner;
+
 public class Duke {
+    public static String addTaskSalutation(String item) {
```

Comment:
printTaskSalutation might be a better method name.

Reply:
It would help to add comments for your method to clarify what it is doing.